### PR TITLE
Update dashboards

### PR DESCRIPTION
**Pull request by kvaps**
Signed-off-by: Andrei Kvapil <kvapss@gmail.com>

## Summary by CodeRabbit

- **New Features**
	- Added a new data source configuration for Prometheus.
	- Introduced new panels for network metrics in Kubernetes dashboards.
	- New "Bar gauge" panel type added to the Kubernetes global views.
	- Enhanced visualizations with new properties for displaying metrics.

- **Bug Fixes**
	- Updated Prometheus expressions to improve data filtering and accuracy.

- **Version Updates**
	- Upgraded Grafana and plugin versions across multiple dashboard configurations.

- **Improvements**
	- Enhanced dashboard layouts and usability with new visualization options.
	- Adjusted configurations for better performance and clarity in monitoring metrics.

### CODE REVIEW

**Comment by gecube** [comment]
Where could I check all the dashboard in the wild or on the cozy-stack instance?

File: dashboards/control-plane/deprecated-resources.json [hunk 1]
```diff
@@ -590,6 +590,25 @@
...
+        "label": "Prometheus",
+        "multi": false,
+        "name": "ds_prometheus",
+        "options": [],
+        "query": "prometheus",
+        "queryValue": "",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
```
[the remark this answers is not in the document]
Comment (themoriarti) [reply dashboards/control-plane/deprecated-resources.json:611]: @kvaps You have duplicate source.

**Review by themoriarti — COMMENTED** [review]
Perhaps such a duplication is needed, but it looks like a duplication

**Comment by kvaps** [comment]
> Where could I check all the dashboard in the wild or on the cozy-stack instance?

You can build your own release and apply it in cluster

https://cozystack.io/docs/development/

**Comment by kvaps** [comment]
Thanks, fixed

**Pull request merged**